tests/installation/install_and_reboot.pm

```diff
@@ -48,6 +48,8 @@ sub run {
     my $self = shift;
     # NET isos are slow to install
     my $timeout = 2000;
+    # aarch64 can be particularily slow depending on the hardware
```

Comment:
Can you be more specific why this would be the case for aarch64? As stated in the other comments: If just the worker is slow then put TIMEOUT_SCALE on the worker config. Otherwise I would assume there is either an architecture specific design issue that prevents more efficient I/O or maybe product bugs we have to keep in mind?